vmware_aria_operations_integration_sdk/adapter_configurations/adapter_config.py

```diff
@@ -0,0 +1,343 @@
+        self.ignore_files = {".DS_Store", "Thumbs.db", "desktop.ini"}
+
+        return content_dir
+
+    def _add_git_keep_file(self, path: str) -> None:
+        with open(os.path.join(path, ".gitkeep"), "w") as gitkeep:
+            # Create empty .gitkeep file
+            pass
+
+    def _create_manifest_localization_file(self) -> None:
+        mkdir(self.conf_resources_dir_path)
+        resources_file = os.path.join(
+            self.conf_resources_dir_path, "resources.properties"
+        )
+        with open(resources_file, "w") as resources_fd:
+            resources_fd.write("#This is the default localization file.\n")
+            resources_fd.write("\n")
+            resources_fd.write("#The solution's localized name displayed in UI\n")
+            resources_fd.write(f"DISPLAY_NAME={self.display_name}\n")
+            resources_fd.write("\n")
+            resources_fd.write("#The solution's localized description\n")
+            resources_fd.write(f"DESCRIPTION={self.adapter_description}\n")
+            resources_fd.write("\n")
+            resources_fd.write("#The vendor's localized name\n")
+            resources_fd.write(f"VENDOR={self.vendor}\n")
+
+    def _create_eula_file(self) -> None:
+        if not self.eula_file_path:
+            self.eula_file_path = os.path.join(self.project.path, "eula.txt")
+            with open(self.eula_file_path, "w") as eula_fd:
+                # Note: VMware Aria Operations requires a EULA file, and it must not be blank.
+                eula_fd.write("There is no EULA associated with this Management Pack.")
+        else:
+            copy(self.eula_file_path, self.project.path)
+
+    def _create_icon_file(self) -> None:
+        if not self.icon_file_path:
+            self.icon_file_path = ""
+        else:
+            copy(self.icon_file_path, self.project.path)
+            self.icon_file_path = os.path.basename(self.icon_file_path)
+
+    def _create_manifest_file(self) -> None:
+        self.manifest_file_path = os.path.join(self.project.path, "manifest.txt")
+        manifest = {
+            "display_name": "DISPLAY_NAME",
+            "name": f"iSDK_{self.adapter_key}",
+            "description": "DESCRIPTION",
+            "version": "1.0.0",
+            "vcops_minimum_version": "8.10.0",
+            "disk_space_required": 500,
+            "run_scripts_on_all_nodes": "true",
+            "eula_file": self.eula_file_path,
+            "platform": ["Linux Non-VA", "Linux VA"],
+            "vendor": "VENDOR",
+            "pak_icon": self.icon_file_path,
+            "pak_validation_script": {"script": ""},
+            "adapter_pre_script": {"script": ""},
+            "adapter_post_script": {"script": ""},
+            "adapters": ["adapter.zip"],
+            "adapter_kinds": [self.adapter_key],
+            "license_type": "",
+        }
+        with open(self.manifest_file_path, "w") as manifest_fd:
+            json.dump(manifest, manifest_fd, indent=4)
```

Comment:
In the future, we should change these (and some of the other files that get generated) to actual files (in, say, a 'default' directory) that get run through the template function.